templates/content/GeoBlazor.Template.Server/GeoBlazor.Template.Server.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
why is there a solution file at this level?

Reply:
it was just convenient for building/testing, I can remove it